### DISCUSSION TITLE
Multiline Markdown links aren't recognized in raw

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#50207] [Mobile] [answered]
GitHub Mobile beta
1. Open  https://github.com/dreamworksanimation/openmoonray/blob/release/CONTRIBUTING.md
2. Use the ⚙️ to select view raw
    ![Screenshot_20230316-051648.png](https://user-images.githubusercontent.com/2119212/225571786-d7819c7c-25bb-4f15-aa33-01a5950f58fa.png)

* Note that singleline (lines are based on line numbers on the left, not lines on the screen) links are "colorized" (I'm using a night view mode, so it's actually just gray in grayscale for me).
* Note that multiline links aren't colorized

**Comment by aconsuegra** [comment, marked as answer]
Thanks for your report! I'll create an internal issue to keep track of it.
